tests/sentry/sentry_metrics/limiters/test_writes_limiter.py

```diff
@@ -24,13 +24,14 @@ def test_writes_limiter_no_limits(set_sentry_option):
     ), set_sentry_option("sentry-metrics.writes-limiter.limits.performance.global", []):
         writes_limiter = get_writes_limiter(PERFORMANCE_PG_NAMESPACE)
 
-        key_collection = KeyCollection(
+        use_case_keys = UseCaseKeyCollection(
             {
-                1: {"a", "b", "c"},
-                2: {"a", "b", "c"},
+                UseCaseID.TRANSACTIONS: {
+                    1: {"a", "b", "c"},
+                    2: {"a", "b", "c"},
+                }
```

Comment:
Can we have some test cases where there are multiple UseCaseIDs in a batch of messages in this test? To verify that the writes limiter is indeed toggling between the different configured write limits for each use case?

Likely for this we'll need to patch the values of some configurations/options and mappings, I'll be doing the analogous testing on the cardinality limiter PR

Reply:
Done